.evergreen/config.yml

```diff
@@ -1452,6 +1463,20 @@ functions:
         file: src/uri-expansions.yml
 
   "run driver test suite":
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        background: true
+        binary: ${PYTHON3}
+        args:
+          - .evergreen/happy-eyeballs-server.py
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        binary: ${PYTHON3}
+        args:
+          - .evergreen/happy-eyeballs-server.py
+          - --wait
```

Comment:
I tried having a "fork background server, wait in foreground" mode to avoid the need for a second command here, and got one that worked for interactive invocations, but Evergreen seems to follow the fork and wait for the child process so that didn't work where it needed to :(